ircbot/plugins/transfer.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+from ircbot import bot
+from redis import StrictRedis
+
+
+@bot.command('transfer')
+def transfer(bot, channel, sender, args):
+    """"command to transfer karma & butts from one term to another"""
+
+    if len(args) == 2:
+            first_name = args[0].lower()
+            second_name = args[1].lower()
+    else:
+        string = args.join(" ")
+        strings = []
+        for term in re.findall("(?![\'\"])[a-z,\s]+(?=[\"\'])", string):
+            strings.append(term.strip())
+        try:
+            first_name = strings[0].lower()
+            second_name = strings[1].lower()
+        except IndexError:
+            bot.message(channel, "Oops! Looks like you didn't give enough arguments!")
+            return
+
+    if sender.lower() == bot.config['System']['owner']:
```

Comment:
IMO, this should come earlier - why perform the searching if the user isn't authorized to run the command?

Reply:
oh, I had that originally but changed it for when I added the permission to be able to transfer your own karma